### DISCUSSION TITLE
No logs available for new Vercel project

### DISCUSSION THREAD

**Discussion by dlwiest** [vercel/community#6010] [Help]
### Summary

Created a new NextJS project and deployed to Vercel about a week ago. Deployment itself seems to be working fine, but I'm not seeing anything in the runtime logs section. I added a console.log to the home screen and confirmed that it's displaying in the terminal, but nothing in the Vercel dashboard. Not seeing logs for GET requests either, and all of the filters are empty. I see logs for a project I deployed a few months ago, and I don't remember needing to configure anything special for that. Do I need to enable something in order for logs to populate?

### Steps to Reproduce

Created a new Vercel project
Populated with Next application via GitHub
Runtime logs are always empty

**Comment by flashlog** [comment]
Facing the same issue.

**Reply by amyegan** [reply]
Thanks for reporting this. It seems related to an [incident](https://www.vercel-status.com/incidents/5z7k3sczh4jl) today that has been closed. Please let me know if you continue to have trouble with runtime logs.

https://www.vercel-status.com/incidents/5z7k3sczh4jl

**Reply by dlwiest** [reply]
I still don't see logs for the app in questions. Older apps seem to work fine.

**Reply by Jooliashi** [reply]
what type of routes are in your deployment? (ie: static, serverless or edge)
